### DISCUSSION TITLE
a few issues I've noted...

### DISCUSSION THREAD

**Discussion by olearydj** [chrisgrieser/shimmering-focus#183] [Q&A] [answered]
A few things I've noticed while getting up to speed on this great theme. Hopefully this is an appropriate place to share them. 

These are all little ticky-tack things. Just sharing in case you weren't aware. There is no way anyone could even attempt to test all the possible combos. Ha!

Obsidian v0.15.6, SF v7-13, Style Settings 0.4.10: [style-settings.json.zip](https://github.com/chrisgrieser/shimmering-focus/files/9097876/style-settings.json.zip)

Dead link indicator in sub-bullet of task in dataview query (read mode):
<img width="359" alt="image" src="https://user-images.githubusercontent.com/42705243/178616858-a6b8109e-61ef-4aed-9f2e-0ed51e922c88.png">

Drop lines in file explorer seem offset right:
<img width="184" alt="image" src="https://user-images.githubusercontent.com/42705243/178617541-b621b351-c7b3-45f2-ae3e-1434478e9ae8.png">

Is there a way to tighten up the spacing in the tag and file explorer panes? It would be nice if this was style as Recent files is.
<img width="148" alt="image" src="https://user-images.githubusercontent.com/42705243/178617280-03f6f28d-63d3-4ead-bb22-6347c0c8c026.png">

Spacing for gutter markers in the footer is tight, especially for numbered version:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/42705243/178618498-bb1f8e53-cf72-4501-bdb0-f158169c804b.png">

Multiple tags in gutter overlap:
<img width="515" alt="image" src="https://user-images.githubusercontent.com/42705243/178618960-5962db18-79e9-4215-9a06-ad7a75383a98.png">

Spoilers render white, do not unhide on hover (even in reading mode):
<img width="135" alt="image" src="https://user-images.githubusercontent.com/42705243/178620333-640fd34d-cb5b-44ea-9bc8-55c745ff34ec.png">

Citation quote formatting in reading mode:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/42705243/178622145-62607693-d115-4ce9-8a68-d3d225863932.png">

**Comment by chrisgrieser** [comment, marked as answer]
hi! For bugs, please create an issue (or better one issue per bug), since the bug report form includes various kinds of information I need to fix this 🙂

**Comment by olearydj** [comment]
Ok, I've converted these to bugs and one FR. Sorry to blow up your inbox!

**Reply by chrisgrieser** [reply]
thanks! and no worries about blowing up my inbox. It just might take me a while to get to it, since I am quite a bit busy till later this summer – I recently fixed all the major issues for the 0.15 release though.
